tests/dns/dns-rcode/test.rules

```diff
@@ -0,0 +1,5 @@
+# Should alert in client direction.
+alert dns any any -> any any (dns.rcode:3; sid:1; rev:1;)
+
+# Should only alert in client direction.
+alert dns any any -> any any (dns.rcode:!3; sid:2; rev:1;)
```

Comment:
This combination is a bit tricky. We want to alert on rcode 3 and also not on rcode3 so basically, all rcodes? 😉

Reply:
got it, I'll change the rule and create a new PR